cpp/dolfinx/mesh/Mesh.cpp

```diff
@@ -413,14 +451,13 @@ mesh::create_submesh(const Mesh& mesh, int dim,
 
   // Create submesh coordinate element
   CellType submesh_coord_cell
-      = cell_entity_type(mesh.geometry().cmap().cell_shape(), dim, 0);
-  auto submesh_coord_ele = fem::CoordinateElement(
-      submesh_coord_cell, mesh.geometry().cmap().degree());
+      = cell_entity_type(geometry.cmap().cell_shape(), dim, 0);
+  auto submesh_coord_ele
+      = fem::CoordinateElement(submesh_coord_cell, geometry.cmap().degree());
 
   // Submesh geometry input_global_indices
   // TODO Check this
-  const std::vector<std::int64_t>& mesh_igi
-      = mesh.geometry().input_global_indices();
+  const std::vector<std::int64_t>& mesh_igi = geometry.input_global_indices();
```

Comment:
Maybe a more instructive name than `mesh_igi`?

Reply:
Could we do that in a separate PR, as I haven't actually changed anything wrt. `mesh_igi`, except of some formatting.